README.md

```diff
@@ -90,3 +90,8 @@ BEGIN
     RETURN c;
 END
 ```
+
+# Import & Export
+
+This is now possible to import or export data in different part of the site. Both of them can be done using CSV, XLS and ODS files.
+Note that during the import part, all beneficiaries you modified in the imported file will be updated. Moreover, if a beneficiary is missing in the distribution but is present in all the beneficiaries of the project, he'll be suppressed from the distribution. There is the same process for a beneficiary added in the imported file. Finally, if you added a beneficiary unknown from all the beneficiaries of the project, he'll be added to a "errors" part to display you that all users in this part won't be added to the distirbution.
```

Comment:
Modify pas modified (on va le faire au présent)
Pas suppressed mais removed ou deleted.
The same process goes for beneficiaries added in the imported file.
if you add a beneficiary that is not part of the project (in the database), ...
added in
array au lieu de part
au lieu de to display blabla, that shows all users that won't be added to...